packages/doc/content/components/getting-started/index.mdx

```diff
@@ -9,16 +9,19 @@ order: 0
 
 Gympass design-system's follows design guidelines specification, we developed a React and React Native UI library that contains a set of high quality components that defines our interfaces.
 
+An important note is that the **styled-component** is a peerDependency [see why]("https://styled-components.com/docs/faqs#i-am-a-library-author-should-i-bundle-styledcomponents-with-my-library"), if you dont have styled-components installed you'll [need to install](https://styled-components.com/docs/basics#installation).
+
 ### Installing
 
 In order to install our design-system just run:
 
 ```
 yarn add @gympass/yoga
 
-// or
+//or
 
 npm install @gympass/yoga
+
```

Comment:
What do you think to remove this breakline? it will add this breakline to the preview on the codeblock component.

```suggestion
```

Reply:
Done